examples/arquillian-jpa/src/test/resources/arquillian.xml

```diff
@@ -25,7 +25,7 @@
     <property name="defaultDataSource">openejb:Resource/demoDataSource</property>
   </extension>
 
-  <container qualifier="tomee" default="true">
+  <container qualifier="tomee-embedded" default="true">
```

Comment:
Looks like it changed from embedded to remote?`If so, we might need to update the `README` as well?

Reply:
I may see it wrong, but I think I’ve switched from remote to embedded. If I use the remote container, the injection of the persistence context and the @EJB annotation are not working (the entitity manager is null and the transactionalCaller is also null

Reply:
You are right. My bad ;)